addons/crm_iap_mine/models/crm_iap_lead_mining_request.py

```diff
@@ -333,12 +334,16 @@ def action_get_lead_action(self):
         self.ensure_one()
         action = self.env["ir.actions.actions"]._for_xml_id("crm.crm_lead_all_leads")
         action['domain'] = [('id', 'in', self.lead_ids.ids), ('type', '=', 'lead')]
+        action['context'] = dict(ast.literal_eval(action.get('context', '{}')), create=False)
         return action
 
     def action_get_opportunity_action(self):
         self.ensure_one()
         action = self.env["ir.actions.actions"]._for_xml_id("crm.crm_lead_opportunities")
         action['domain'] = [('id', 'in', self.lead_ids.ids), ('type', '=', 'opportunity')]
+        context = action.get('context', '{}')
+        context = context.replace("'default_user_id': uid", f"'default_user_id': {self.env.uid}")
```

Comment:
Counter-attack, how about adding `context="{'create': False}"` on the XML button that calls this action?

I tested locally and it seems to work, seems better if we can avoid this hack ^^

Reply:
Nice! Yes it feels better than a ugly hack :p I've done the change. I see that there are almost no occurrences of [context on button with no_create] in the code, so I kept the other action as is, but I can also use it there if you prefer?